tests/discovery/test_url_to_graph.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from conftest import run_single_test
+from rdflib import Graph
+
+from sema.discovery.url_to_graph import (
+    ctype_to_rdf_format,
+    get_graph_for_format,
+)
+
+log = logging.getLogger(__name__)
+
+
+def test_ctype_to_rdf_format():
+    assert ctype_to_rdf_format("application/ld+json") == "json-ld"
+    assert ctype_to_rdf_format("text/turtle") == "turtle"
+    assert ctype_to_rdf_format("application/json") is None
+    assert ctype_to_rdf_format("application/octet-stream") is None
+    assert ctype_to_rdf_format("text/html") is None
+
+
+# ttl file
+# jsonld file
+# html file with ttl file in head as full script
+# html file with jsonld file in head as script link
+test_cases = [
```

Comment:
these tests and assertions heavily depend on external services
I would rather see us push up a local http server during tests to better control their outcome 
no need to that now, but best to track an issue for this

Reply:
Issue taken up in #66